bin/tests/check_dl_all.sh

```diff
@@ -14,3 +14,7 @@ bash ./bin/tests/check_dl_cv.sh
 
 ####################################  NLP  ####################################
 bash ./bin/tests/check_dl_nlp.sh
+
+
+#################################  Callbacks  #################################
+bash ./bin/tests/check_dl_callbacks.sh
```

Comment:
how do these test differ from https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_core_callbacks.sh ?

Reply:
and could we rewrite them into `check_dl_core{something}.sh`, so we could automatically use them https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_all.sh#L8 ?

Reply:
> how do these test differ from https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_core_callbacks.sh ?

They are the same cause I created a branch from branch used in #784.

> and could we rewrite them into `check_dl_core{something}.sh`, so we could automatically use them https://github.com/catalyst-team/catalyst/blob/master/bin/tests/check_dl_all.sh#L8 ?

Sure, I'll update file name.